src/main/java/helper/command/AddCommand.java

```diff
@@ -0,0 +1,68 @@
+package helper.command;
+
+import helper.DukeException;
+import helper.Storage;
+import helper.TaskList;
+import helper.Ui;
+import task.Deadline;
+import task.Event;
+import task.Todo;
+
+import java.time.LocalDate;
+
+/**
+ * Commands for adds (todo..., deadline..., event...)
+ */
+public class AddCommand extends Command {
+
+    private final String whichAdd;
+    private final String addString;
```

Comment:
addString sounds like a verb instead of a noun. This string sounds like a method instead of a variable.
Edit: okay after reading more of your code, I am thinking this naming might not be wrong.